cli/command/volume/create.go

```diff
@@ -72,7 +72,7 @@ func runCreate(dockerCli *command.DockerCli, opts createOptions) error {
 }
 
 var createDescription = `
-Creates a new volume that containers can consume and store data in. If a name
+Create a new volume that containers can consume and store data in. If a name
```

Comment:
You're not creating the volume, the command is. I think it needs to stay how it was.